src/components/FormSelect/FormSelect.tsx

```diff
@@ -38,9 +43,19 @@ export const FormSelect = forwardRef(
         <option disabled value="">
           {defaultText}
         </option>
-        {options.map(option => (
-          <option key={`option-${option}`}>{option}</option>
-        ))}
+        {options.map(option => {
+          if (typeof option === 'object') {
+            return (
+              <option
+                key={`option-${option.label}`}
+                value={option.value || option.label}
```

Comment:
Are there likely to be any instances where there is a label but no value?

Reply:
probably not. I will update.